client/src/pages/schema/Types.ts

```diff
@@ -78,3 +78,21 @@ export interface IndexExtraParam {
   metric_type: string;
   params: string;
 }
+
+export interface SizingToolParam {
+  // type is float or bytes
+  type: string;
```

Comment:
type: float | bytes is better ?
Where did you use it ? I can't find it in this pr.